src/java/htsjdk/samtools/Defaults.java

```diff
@@ -73,6 +79,7 @@
             NON_ZERO_BUFFER_SIZE = BUFFER_SIZE;
         }
         REFERENCE_FASTA = getFileProperty("reference_fasta", null);
+        REF_DOWNLOAD_ENABLED = false;
```

Comment:
You need to call `getBooleanProperty("ref_download_enabled", false)` here to allow the default of false to be overridden by a system property.